core/src/main/java/org/kiwix/kiwixmobile/core/help/HelpScreen.kt

```diff
@@ -0,0 +1,205 @@
+  }
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun HelpTopAppBar(navController: NavController) {
+  // Retrieve the actionBarSize from the current theme
+  val context = LocalContext.current
+  val actionBarHeight = with(LocalDensity.current) {
+    // Obtain the height defined in the theme (usually 56dp on phones)
+    val styledAttributes =
+      context.theme.obtainStyledAttributes(intArrayOf(android.R.attr.actionBarSize))
+    styledAttributes.getDimension(0, 0f).toDp().also { styledAttributes.recycle() }
+  }
+
+  TopAppBar(
+    modifier = Modifier.height(actionBarHeight), // set the height here
+    title = {
+      Row(
+        modifier = Modifier.fillMaxSize(),
+        verticalAlignment = Alignment.CenterVertically
+      ) {
+        Text(
+          modifier = Modifier.padding(
+            start = dimensionResource(R.dimen.activity_horizontal_margin)
+          ),
+          text = stringResource(id = R.string.menu_help),
+          color = Color.White,
+          fontWeight = FontWeight.SemiBold
+        )
+      }
+    },
+    navigationIcon = {
+      Row(modifier = Modifier.fillMaxHeight(), verticalAlignment = Alignment.CenterVertically) {
+        IconButton(onClick = navController::popBackStack) {
+          Icon(
+            imageVector = Icons.AutoMirrored.Filled.ArrowBack,
+            contentDescription = "Back_Navigation",
+            tint = Color.White
+          )
+        }
+      }
+
+    },
+    colors = TopAppBarDefaults.topAppBarColors(
+      containerColor = Color.Black
+    )
+  )
+}
+
+@Composable
+fun HelpContent(
+  data: List<HelpScreenItemDataClass>,
+  dividerColor: Color,
+  innerPadding: androidx.compose.foundation.layout.PaddingValues
+) {
+  Column(
+    modifier = Modifier
+      .padding(innerPadding)
+  ) {
+    SendReportRow()
+    HelpItemList(data, dividerColor)
+  }
+}
+
+@Composable
+fun SendReportRow() {
+  val context = LocalContext.current
+  val isDarkTheme = isSystemInDarkTheme()
+
+  Row(
+    modifier = Modifier
+      .fillMaxWidth()
+      .clickable {
+        (context as? Activity)?.start<DiagnosticReportActivity>()
+      },
+    verticalAlignment = Alignment.CenterVertically,
+    horizontalArrangement = Arrangement.Start
+  ) {
+    Image(
+      painter = painterResource(R.drawable.ic_feedback_orange_24dp),
+      contentDescription = stringResource(R.string.send_report),
+      modifier = Modifier
+        .padding(dimensionResource(R.dimen.activity_horizontal_margin))
+    )
+
+    Text(
+      text = stringResource(R.string.send_report),
+      color = if (isDarkTheme) Color.LightGray else Color.DarkGray,
+      fontSize = SendDiagnosticReportFontSize
+    )
+  }
+}
+
+@Composable
+fun HelpItemList(data: List<HelpScreenItemDataClass>, dividerColor: Color) {
+  LazyColumn(
+    modifier = Modifier
+      .fillMaxWidth()
+  ) {
+    itemsIndexed(data, key = { _, item -> item.title }) { _, item ->
+      HorizontalDivider(
```

Comment:
You should add this divider after the `HelpScreenItem` so that it will be below the content, and remove the last extra item added here(The below code) to simplify the design and code.

```kotlin
item {
      HorizontalDivider(
        color = dividerColor
      )
    }
```